### ISSUE TITLE
How to use on MacOS?

### ISSUE THREAD

**Issue by HowcanoeWang** [lmbelo/python3-embeddable#4]
I have downloaded and unzipped, but unlike windows, there is no python.exe at root folder available.

then I tried to run the `activate.sh` by:

```bash
python3-macos-3.9.12-universal2 % ./activate.sh
```

Nothing happend.

And also tried to run:

```bash
python3-macos-3.9.12-universal2 % ./bin/python3.9
```

After giving software permissions, I got the following error:

```bash
dyld[2909]: Library not loaded: '/usr/local/lib/libpython3.9.dylib'
  Referenced from: '/Users/name/Downloads/python3-macos-3.9.12-universal2/bin/python3.9'
  Reason: tried: '/usr/local/lib/libpython3.9.dylib' (no such file), '/usr/lib/libpython3.9.dylib' (no such file)
zsh: abort      ./bin/python3.9
```

Any ideas?

**Comment by HowcanoeWang** [comment]
solved by the follow method, sorry for the lack of basic background about using the CLI;

```bash
python3-linux-3.10.4-x86_64 % source activate.sh
python3-linux-3.10.4-x86_64 % python3.10 -m pip list
Package    Version
---------- -------
pip        22.0.4
setuptools 58.1.0
```

**Comment by lmbelo** [comment]
You must type "source" or "." ./activate.sh to export settings to your current terminal.

**Comment by HowcanoeWang** [comment]
> You must type (under source) . ./activate.sh to export settings to your current terminal.

Could you kindly add this point in the readme file? Say how to use this tool on different platforms for greenhands.

**Comment by lmbelo** [comment]
Yes, sure... Fyi, when you call a script from terminal, a new process is created, so any environment variable created in that script won't be available in your terminal at the end of execution. If you place a "." or the "source" command before your script file, you're telling the terminal to run your script in the current terminal instead, making all environment variable set in that script available to your current terminal. Did you get the point?

In short, you were not using the desired Python environment, but one installed in your system.

**Comment by lmbelo** [comment]
Have a look [here](https://github.com/lmbelo/python3-embeddable/blob/bb54f84b54775b832215e187456f3642fe524917/python3-linux/build.sh#L50).

**Comment by HowcanoeWang** [comment]
> Yes, sure... Fyi, when you call a script from terminal, a new process is created, so any environment variable created in that script won't be available in your terminal at the end of execution. If you place a "." before your script file, you're telling the terminal to run your script in the current terminal instead, making all environment variable set in that script available to your current terminal. Did you get the point?
> 
> In short, you were not using the desired Python environment, but one installed in your system.

Thanks for this detailed explanation! Got this point and this indeed a very basic unix skills I did not know before.
